driver-core/src/main/com/mongodb/client/model/Aggregates.java

```diff
@@ -173,6 +174,25 @@ public static Bson unwind(final String fieldName) {
         return new BsonDocument("$unwind", new BsonString(fieldName));
     }
 
+    /**
+     * Creates a $unwind pipeline stage for the specified field name, which must be prefixed by a {@code '$'} sign.
+     *
+     * @param fieldName the field name, prefixed by a {@code '$' sign}
+     * @param unwindOptions options for the unwind pipeline stage
+     * @return the $unwind pipeline stage
+     * @mongodb.driver.manual reference/operator/aggregation/unwind/ $unwind
+     * @since 3.2
+     */
+    public static Bson unwind(final String fieldName, final UnwindOptions unwindOptions) {
```

Comment:
Now that I understand this better (see above), the @mongodb.server.release 3.2 makes sense and should be here, as this way of using $unwind will fail if server < 3.2